gems/sorbet-runtime/lib/types/props/custom_type.rb

```diff
@@ -1,20 +1,35 @@
 # frozen_string_literal: true
-# typed: false
+# typed: strict
 
 module T::Props
   module CustomType
+    extend T::Sig
+    extend T::Helpers
+
+    abstract!
+
     include Kernel # for `is_a?`
 
-    # Returns true if the given Ruby value can be assigned to a T::Props field
-    # of this type.
-    #
-    # @param [Object] _value
-    # @return T::Boolean
-    def instance?(_value)
-      raise NotImplementedError.new('Must override in included class')
+    # Alias for backwards compatibility
+    sig(:final) do
+      params(
+        value: BasicObject,
+      )
+      .returns(T::Boolean)
+      .checked(:never)
```

Comment:
Huh just wanted to check that `.checked(:never)` didn't affect `:final` and looks like Sorbet does correctly still raise for this code at runtime:

```ruby
# typed: true

require 'sorbet-runtime'

T::Configuration.enable_final_checks_on_hooks


class Module
  include T::Sig
end

class Parent
  sig(:final) {void.checked(:never)}
  def foo; end
end

class Child < Parent
  def foo; end
end

Child.new.foo
```